### ISSUE TITLE
The furnace not a11y

### ISSUE THREAD

**Issue by dominikwilkowski** [govau/designsystem#152]
Yo DS friends :)

So turns out [WCAG 2.1 now has a requirement](https://www.w3.org/WAI/GL/wiki/WCAG_2.1_Implementations#3.2.6_Status_Changes) on aria-live regions and that's the tiny bit missing from the furnace page https://designsystem.gov.au/download... Should have the codebox in `.js-furnace-code` marked up as a polite region imho

Love
❤️

**Comment by alex-page** [comment]
Thanks @dominikwilkowski <3

**Comment by pattyde** [comment]
Appreciate the heads up Dom :)

**Comment by pattyde** [comment]
Fixed by https://github.com/govau/designsystem/issues/154

**Comment by dominikwilkowski** [comment]
Actually it looks like it was fixed by #153 and #154 was just the merge to master @pattyde ❤️ 
hahaaa
